File: RestaurantManagementApp/src/app/shared/inventory-history.model.ts
export class InventoryHistoryModel {
  public Id: string;
  public InventoryId: string;
  public UpdatedQuantity: number;
  public UpdateTime: string;
  public Unit: string;
  public CurrentPrice: number;

  constructor(id: string, inventoryId: string, updatedQuantity: number,
              updateTime: string, unit: string, currentPrice: number) {
    this.Id = id;
    this.InventoryId = inventoryId;
    this.UpdatedQuantity = updatedQuantity;
    this.UpdateTime = updateTime;
    this.Unit = unit;
    this.CurrentPrice = currentPrice;
  }
 }

File: RMS(Server)/RMS(Server)/Models/InventoryHistoryModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace RMS_Server_.Models
{
    public class InventoryHistoryModel
    {
        public string Id { get; set; }
        public Inventory Inventory { get; set; }
        public string InventoryId { get; set; }
        public int UpdatedQuantity { get; set; }
        public string UpdateTime { get; set; }
        public string Unit { get; set; }    
    }
}